src/Handler/SentryHandler.php

```diff
@@ -86,7 +86,7 @@ protected function write(array $record) : void
 
         if (
                 isset($record['context']['exception']) &&
-                $record['context']['exception'] instanceof \Throwable
+                $record['context']['exception'] instanceof Throwable
```

Comment:
I have this feeling that "arguments" passed to `instanceof` needn't be FQCN...(I've always wondered, if that's true, how are class-paths resolved...*ponder*...)

Reply:
Sorry I just ran an auto-cleanup tool over my code (does things like prefer imports over FQN).

Reply:
As I recall, `instanceof` doesn't trigger autoloading, but using the reference with/without import is purely a stylistic choice and has no real difference in behaviour. :)

As long as you either use the `\` prefix, or you import it (but not both or neither) it's fine.